navigator-menu/src/components/tips-calculator-components/right-portion/RightContainer.jsx

```diff
@@ -0,0 +1,41 @@
+import Amount from './amount-component/Amount';
+import { Container, Wrapper, Reset } from './RightContainer.styles';
+import { useContext } from 'react';
+import { TipsCalculatorContext } from '../../../contexts/TipsCalculatorContext';
+
+function RightContainer() {
+  const { tipsCalc, reset } = useContext(TipsCalculatorContext);
+
+  const totalTip = tipsCalc.bill * tipsCalc.percentage;
+  const tipPerPerson = (totalTip / tipsCalc.people).toFixed(2);
+  const perPersonBill = (
+    tipsCalc.bill / tipsCalc.people +
+    totalTip / tipsCalc.people
+  ).toFixed(2);
+
+  const activeReset = 'bg-tipsCalcResetBtnActive';
+
+  return (
+    <Container>
+      <Amount name="Tip Amount" amount={tipPerPerson} />
+      <Amount name="Total" amount={perPersonBill} />
+      <Wrapper>
+        <Reset
+          onClick={reset}
+          id="reset-btn"
+          className={
+            tipsCalc.bill != 0 ||
```

Comment:
While there is nothing wrong with this code, test removing `!= 0` from the checks. If the number is 0 it's still falsey, you don't need explicit check of the value.